src/package/nuspec/Microsoft.TestPlatform.Portable.nuspec

```diff
@@ -0,0 +1,466 @@
+﻿<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.TestPlatform.Portable</id>
+    <version>15.0.0</version>
+    <title>Microsoft.TestPlatform.Portable</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>The cross platform Microsoft Test Platform.</description>
+    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</licenseUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <projectUrl>https://github.com/microsoft/vstest/</projectUrl>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+  </metadata>
+  <files>
+    <!-- Add a third party notice file -->
+    <file src="ThirdPartyNotices.txt" target="" />
+    
+    <!-- net451 -->
+    <file src="net451\$Runtime$\datacollector.exe"	target="tools\net451\datacollector.exe" />
```

Comment:
Do we require runtime for portable? The binaries will execute with Mono as well.